boards/native/include/board.h

```diff
@@ -127,6 +127,7 @@ extern mtd_dev_t *mtd0;
 /** @} */
 #endif
 
+#ifdef MODULE_MOTOR_DRIVER
```

Comment:
Would have preferred an `IS_USED` but not worth re-triggering the ci for this.

Reply:
Ah well build failed, so this could be addressed

Reply:
yup.